src/asset.js

```diff
@@ -44,8 +45,8 @@ export class Asset {
 
   /**
    * Returns an asset object from its XDR object representation.
-   * @param {xdr.Asset} assetXdr - The asset xdr object.
-   * @returns {Asset}
+   * @param {xdr.Asset | xdr.ChangeTrustAsset} assetXdr - The asset xdr object.
```

Comment:
This, no?
```suggestion
   * @param {xdr.Asset | xdr.ChangeTrustAsset | xdr.TrustLineAsset} assetXdr - The asset xdr object.
```

Reply:
This is incomplete, TrustLineAsset is not inheriting from Asset yet. I'm waiting for @.Shaptic to reply in https://github.com/stellar/js-stellar-base/pull/454#discussion_r694852344 before doing that.